templates/docs/docs/conf.py.j2

```diff
@@ -317,3 +317,8 @@ class MyPythonDomain(PythonDomain):
 def setup(sphinx):
     """Use MyPythonDomain in place of PythonDomain"""
     sphinx.add_domain(MyPythonDomain, override=True)
+
+rst_prolog = """.. attention::
+    This docs will be deactivated in July 2024. `Learn More <https://discourse.pulpproject.org/t/unified-documentation-part-2/1188>`_
```

Comment:
Let's just say "This documentation will be deactivated in the near future." so we don't have to do anything on a specific date.

Reply:
Yep. Never write docs that plan to be outdated.